ground/gcs/src/plugins/uavobjects/uavobjecttemplate.m

```diff
@@ -30,6 +30,7 @@
 str5=[];
 multipleInstanceLookup = zeros(0,2);
 overo = false;
+onboardLogger = true;
```

Comment:
Whoops, totally! I knew there was a reason why I hadn't pushed this PR several months ago. I need to consider how the Matlab logger can more gracefully, or even automatically, select log file types.